apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -541,11 +541,18 @@ Shared.Background {
                     height: searchField.height
                 }
 
-                model: BookmarkModel {
+                BookmarkModel {
                     id: bookmarkModel
                     activeUrl: toolBar.url
                 }
 
+                model: BookmarkFilterModel {
+                    id: bookmarkFilterModel
+                    sourceModel: bookmarkModel
+                    maxDisplayedItems: search ? favoriteGrid.columns : bookmarkModel.count
+                    search: searchField.text !== webView.url ? searchField.text : ""
+                }
```

Comment:
When searching, the `FavoriteGrid` is immediately hidden. Should there be some change to [`historyContainer.showFavorites` on line 223](https://github.com/sailfishos/sailfish-browser/pull/798/files#diff-2ddfbf1787333eb21930f77d91dc8b92fc930d6dd5eb8d801df0c9499fe06bf8R223) to avoid this?